include/sensor_plugin.h

```diff
@@ -57,4 +57,6 @@ class SensorPlugin {
   JSBSim::FGFDMExec *_sim_ptr;
   double _last_sim_time;
```

Comment:
Better initialize the things you can do here (less code):
```suggestion
  double _last_sim_time{0.};
```